pkg/machines/xmlCreator.js

```diff
@@ -94,7 +94,7 @@ export function getNetworkXML({ name, forwardMode, device, ipv4, netmask, ipv6,
         const hostElem = doc.createElement('host');
         hostElem.setAtrribute('ip', ipv4);
         const hostnameElem = doc.createElement('hostname');
-        const hostnameTextNode.doc.createTextNode('gateway');
+        const hostnameTextNode = doc.createTextNode('gateway');
```

Comment:
Could you please fix this typo in a commit where you introduced it? Since it's in same Pull request, there is no need for typos made in one commit to be fixed another commit. (rebase and fixup should do the job)

Reply:
Done. Thanks for the hint, I didn't know before how to do that.